docs: Update README stating this project is in maintenance mode

jodh-intel opened the pull request:
Now that we have released [Clear Containers
3.0](https://github.com/clearcontainers/runtime/releases/tag/3.0.0),
this codebase (CC 2.x) is now deprecated and the project is in
maintenance mode.

Updated the `README.rst` to state this and encourage users to switch to
the [new runtime](https://github.com/clearcontainers/runtime).

Fixes #1064.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

Commit:
docs: Update README stating this project is in maintenance mode

Now that we have released [Clear Containers
3.0](https://github.com/clearcontainers/runtime/releases/tag/3.0.0),
this codebase (CC 2.x) is now deprecated and the project is in
maintenance mode.

Updated the `README.rst` to state this and encourage users to switch to
the [new runtime](https://github.com/clearcontainers/runtime).

Fixes #1064.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

jodh-intel commented:
ping @jcvenegas, @sameo, @grahamwhaley.

grahamwhaley commented:
The words look fine - but the file is not rendering for me when viewed on github on your branch. I suspect it should, so wonder if somehow something has broken the rst'ness of the file - can you check that for me?

grahamwhaley commented:
and you'll need @iphutch as well.

jcvenegas commented:
@jodh-intel hey james,  looks good :) , but also I could not seen the text well rendered, even taking a look at your branch directly in your fork. I wonder if the `warning` directive is not recognized by github render.

jodh-intel commented:
Branch updated.

Hi @iphutch - can you review please?

grahamwhaley commented:
yay, that renders for me now.
lgtm

Commit:
docs: Rename original README

The main project README needs to be renamed to allow a stub README to be
created that will explain succinctly that the project is in maintenance
mode (but provide a link to the original README).

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

Commit:
docs: Replace original README with warning

The main README now contains a warning that this project is in
maintenance mode, along with a link to the original (full) README.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

jodh-intel commented:
Sorry - I had a better idea - branch updated again ;)

grahamwhaley commented on README.rst:
```diff
@@ -1,448 +1,23 @@
-.. [#oci-runtime-cli-clarification]
-   https://github.com/opencontainers/runtime-spec/issues/434
+To see the original project ``README``, click
+`here <https://github.com/01org/cc-oci-runtime/blob/master/README-original.rst>`_.
```
nit: you should be able to make this a relative reference like [this](README-original.rst) (or the rst equiv?) I would think? Means it is maybe more future proof (and would de-ref correctly in your pending PR ;-)

jodh-intel replied:
Agreed. Branch de-nitted! ;)

grahamwhaley reviewed (approved):
Yeah, that's a better idea.
tiny nit
lgtm

jodh-intel commented:
@jcvenegas - could you review please?

jodh-intel commented:
or @devimc ;)

jodh-intel commented:
Hi @iphutch - please can you review this so we can land it?
